x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -23,8 +23,8 @@ export const EndpointActionLogRequestSchema = {
   query: schema.object({
     page: schema.number({ defaultValue: 1, min: 1 }),
     page_size: schema.number({ defaultValue: 10, min: 1, max: 100 }),
-    start_date: schema.maybe(schema.string()),
-    end_date: schema.maybe(schema.string()),
+    start_date: schema.string(),
```

Comment:
Just FYI - I think we should keep this, bug want to mention that this could be viewed as a backwards breaking change, since we are now making these two API params required